src/selectors/__tests__/expandThoughts.ts

```diff
@@ -823,3 +824,27 @@ describe('=done', () => {
     expect(isContextExpanded(stateNew, ['a', 'd'])).toBeFalsy()
   })
 })
+
+describe('multicursor', () => {
```

Comment:
Let's also add a test for the clearMulticursors case.

Reply:
I think we need to add puppeteer test for expandThoughts. (Because we need to add keydown event)

Reply:
A puppeteer test is a little heavy for this, but you could add an RTL test with a test Redux store.

Reply:
Ok, I will check about that.

Reply:
e.g. https://github.com/cybersemics/em/blob/0d71558434e9a7dbe09b9396c440084950a87045/src/commands/__tests__/cursorNext.ts